Remove pip_system_certs import as Void provides this in python3-certifi
--- a/qt/aqt/__init__.py
+++ b/qt/aqt/__init__.py
@@ -16,13 +16,6 @@ if "ANKI_FIRST_RUN" in os.environ:
 
     first_run_setup()
 
-try:
-    import pip_system_certs.wrapt_requests
-except ModuleNotFoundError:
-    print(
-        "Python module pip_system_certs is not installed. System certificate store and custom SSL certificates may not work. See: https://github.com/ankitects/anki/issues/3016"
-    )
-
 if sys.version_info[0] < 3 or sys.version_info[1] < 9:
     raise Exception("Anki requires Python 3.9+")
